site/profile/manifests/windows/baseline.pp

```diff
@@ -1,14 +1,13 @@
-class profile::windows::baseline {
+class profile::baseline {
 
-  class { 'chocolatey':
-    notify => Reboot['afterchocolatey'],
+  exec { 'rename-admin':
```

Comment:
Unfortunately this will break other systems applying this baseline, including Domain Controllers.  Can you move this to a different baseline e.g. wsus-baseline?